src/container_info.h

```diff
@@ -112,6 +114,7 @@ class container_info {
         host_info->m_id = HOST_CONTAINER_ID;
         host_info->m_full_id = HOST_CONTAINER_ID;
         host_info->m_name = HOST_CONTAINER_ID;
+        host_info->m_type = CT_HOST;
```

Comment:
Added a "fake" CT_HOST type that resolves to "host" string; same we do today in libs: https://github.com/falcosecurity/libs/blob/master/userspace/libsinsp/sinsp_filtercheck_container.cpp#L460